hack/update-model.sh

```diff
@@ -1,6 +1,29 @@
 #!/bin/bash -e
 
-# This script ensures that all the api-model submodules used across the project are updated to the latest verison.
+# ==============================================================================
+# Script: update-model.sh
+#
+# This script ensures that all the `ocm-api-model` submodules used across the
+# project are updated to the latest version or pinned to a specific commit SHA.
+#
+# USAGE:
+#   ./update-model.sh [COMMIT_SHA]
```

Comment:
can we add a line on the doc about this https://github.com/openshift-online/ocm-sdk-go/blob/979171be01871fe6b1aaf4cca8409f30422e1d8d/CONTRIBUTING.md#updating-the-ocm-sdk ?